src/compas_viewer/_viewer.py

```diff
@@ -0,0 +1,41 @@
+import sys
+
+from PySide6.QtWidgets import QApplication
+
+from compas_viewer.components.renderer import Renderer
+from compas_viewer.config import Config
+from compas_viewer.configurations import ControllerConfig
+from compas_viewer.configurations import RendererConfig
+from compas_viewer.controller import Controller
+from compas_viewer.scene.scene import ViewerScene
+from compas_viewer.ui.ui import UI
+
+
+class Viewer:
+    _instance = None
+
+    def __new__(cls, *args, **kwargs):
+        if not cls._instance:
+            cls._instance = super().__new__(cls)
+            cls._is_initialised = False
+        return cls._instance
+
+    def __init__(self, *args, **kwargs):
+        if not self._is_initialised:
+            if not QApplication.instance():
+                self.app = QApplication(sys.argv)
+            else:
+                self.app = QApplication.instance()
+
+            self.config = Config.from_json("src/compas_viewer/config.json")
+            self.scene = ViewerScene(name="ViewerScene", context="Viewer")
```

Comment:
I feel these arguments are not even necessary, they can be directly hard-written values inside the class itself. so:
```python
self.scene = ViewerScene()
```